setup.py

```diff
@@ -88,7 +88,7 @@ def run(self) -> None:  # pylint: disable=no-self-use
     "license": "MIT",
     "keywords": "raiden ethereum blockchain",
     "install_requires": requirements,
-    "setup_requires": _get_single_requirement(requirements, "py-solc"),
+    "setup_requires": requirements,
```

Comment:
Do we need `setup_requires` at all, if it is the same as install_requires?

Reply:
I tried it out locally and it doesn't seem to work if omitted completely.

Reply:
I have problems finding more information about it other than that it is "deprecated in favor of the PEP 517". If it works, let's keep it for now.